CHANGELOG.md

```diff
@@ -64,6 +37,34 @@ preflightTest.on('completed', (report: PreflightTestReport) => {
   console.log(" Your estimated quality score was " + report.qualityScore);
 });
 ```
+
+The [PreflightTestReport](https://sdk.twilio.com/js/video/releases/2.17.0/docs/global.html#PreflightTestReport) generated by `preflightTest` on `completed` provides statistics that can be used to determine the quality of the connection. Some of the useful stats and their optimal values are listed below.
+
+| Stat                     |  Description | Optimal values
+| -------------------------|--------------|------------------|
+| jitter                   | Packets delay variation on audio tracks | < 30ms
+| packetLoss               | Packet loss as a percent of total packets sent | < 1%
+| rtt                      | Round trip time in milliseconds. | < 200ms
+
+The API also calculates an overall `qualityScore` using jitter, packet loss, and rtt values observed. The quality score can be one of:
+
+| Value                    | Meaning
+| -------------------------|--------------------------------|
+| 0                        | Failed
+| 1                        | Degraded
+| 2                        | Fair
+| 3                        | Good
+| 4                        | Great
+| 5                        | Excellent
+
+
+A few things to note:
+
+- This function uses <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API"> web audio API's.</a> Browser's autoplay policies sometimes require user action before accessing these APIs. Please ensure that this API is called in response to user action like a button click.
+- `preflightTest` emits a `failed` event to indicate test failures. You can use the Preflight `progress` events to better understand where the test failed and refer to
```

Comment:
@makarandp0 I wanted to link to documentation outlining the progress events and noticed that the possible values are not documented here: https://sdk.stage.twilio.com/js/video/releases/2.17.0-rc2/docs/global.html#PreflightProgress

Reply:
good point, I will add those in this PR.

Reply:
I have fixed this in https://github.com/twilio/twilio-video.js/pull/1538/commits/2b2cf0e436612378748f73ec6cb3d796545cd3e1 
Basically it turned out that jsdocs does not like typescript enum - as those are stripped from the generated code. Converted it to const like rest of the code does. With this you should see those in generated doc [here](https://49575-46595751-gh.circle-artifacts.com/0/dist/docs/global.html#PreflightProgress)